Src/Support/Google.Apis.Auth/GoogleJsonWebSignature.cs

```diff
@@ -229,6 +321,69 @@ public class Payload : JsonWebSignature.Payload
             /// </summary>
             [Newtonsoft.Json.JsonPropertyAttribute("prn")]
             public string Prn { get; set; }
+
+            /// <summary>
+            /// The hosted GSuite domain of the user. Provided only if the user belongs to a hosted domain.
+            /// </summary>
+            [Newtonsoft.Json.JsonPropertyAttribute("hd")]
```

Comment:
Can we replace all of these fully-qualified attribute names with `[JsonProperty("...")]` or does that collide with something?